content/pt-br/serverless.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: Serverless
+Status: Completed
+Category: Technology
+---
+
+## O que é
+
+Serverless é um modelo de desenvolvimento nativo na nuvem que permite que desenvolvedores criem aplicações sem que seja necessário gerenciar servidores. Ainda existem servidores no modelo serverless, mas eles são [abstraídos](/abstraction) do desenvolvimento. Um provedor de computação em nuvem se encarrega do provisionamento, manutenção e [escala](/scalability/) da infraestrutura de servidores. Desenvolvedores podem simplesmente carregar o código em [containers](/container/) para implantação. Uma vez implantadas, aplicações serverless respondem automaticamente à demanda, escalando para cima e para baixo conforme necessário. O preço cobrado por provedores de computação em nuvem para funções serverless geralmente é realizada com base na sua utilização sob demanda. Como resultado disto, quando uma função serverless não está em execução ela não tem custo algum.
+
+## Problema relacionado
+
+Em um modelo de [computação em nuvem](/cloud_computing/) com [Infraestrutura-como-Serviço (IaaS)](/infrastructure_as_a_service), usuários adquirem unidades de capacidade, o que significa que o provedor da nuvem fornece componentes de servidores sempre disponíveis para a execução de aplicações. É responsabilidade do usuário fazer com que os servidores escalem para cima durante períodos de alta demanda e que escalem para baixo quando a capacidade fornecida não é mais necessária. A infraestrutura de nuvem utilizada para executar uma aplicação fica, no entanto, ativa mesmo quando a aplicação não está sendo utilizada.
```

Comment:
nit: adding trailing slash to hyperlinks will remove unnecessary redirects
(`https://glossary.cncf.io/infrastructure_as_a_service` → `https://glossary.cncf.io/infrastructure_as_a_service/`).

```suggestion
Em um modelo de [computação em nuvem](/cloud_computing/) com [Infraestrutura-como-Serviço (IaaS)](/infrastructure_as_a_service/), usuários adquirem unidades de capacidade, o que significa que o provedor da nuvem fornece componentes de servidores sempre disponíveis para a execução de aplicações. É responsabilidade do usuário fazer com que os servidores escalem para cima durante períodos de alta demanda e que escalem para baixo quando a capacidade fornecida não é mais necessária. A infraestrutura de nuvem utilizada para executar uma aplicação fica, no entanto, ativa mesmo quando a aplicação não está sendo utilizada.
```